README.md

```diff
@@ -39,8 +41,110 @@ to produce a PDF file.
 Rendered versions will be released periodically at this Github site;
 unless you are intending to contribute or want to render your own
 version, the rendered releases are probably the best option for you.  The
-releases are at 
+releases are at
 <a href=https://github.com/arbor/sp-rest-api-cookbook/releases>https://github.com/arbor/sp-rest-api-cookbook/releases</a>
 
-In addition, there is a rendered HTML version available at 
+In addition, there is a rendered HTML version available at
 <a href=https://arbor.github.io/sp-rest-api-cookbook/sp-rest-api-tutorial.html>https://arbor.github.io/sp-rest-api-cookbook/sp-rest-api-tutorial.html</a>.
+
+## Working with Github forks, branches, and pull requests
+
+1.  Create an account on GitHub that you want to use for this work, or
+    use an existing one.
+2.  Log in to <http://github.com> as that account, go to
+    <https://github.com/arbor/sp-rest-api-cookbook>, and click the
+    Fork button in the upper right corner of the page. Alernatively
+    use the [hub](https://hub.github.com/) git wrapper and Github
+    client and type:
+
+    ```sh
+    hub clone git@github.com:arbor/sp-rest-api-cookbook.git
+    cd sp-rest-api-cookbook
+    hub fork
+    ```
+3.  Copy your fork to your computer by cloning it:
+
+    ```sh
+    git clone git@github.com:<YOUR_GITHUB_ACCOUNT_NAME>/sp-rest-api-cookbook.git
+    cd sp-rest-api-cookbook
+    ```
+4.  Add a remote called 'upstream' to keep up with the version of this
+    at Arbor's github site:
+
+    ```sh
+    git remote add upstream git@github.com:arbor/sp-rest-api-cookbook.git
+    ```
+5.  Create a git branch and switch to it:
+
+    ```sh
+    git checkout -b <branchname>
+    ```
+6.  Make your edits to `sp-rest-api-cookbook.txt`, add files to
+    `code-examples` and `images`, etc.
+7.  To track your work you can make as many commits as you want, they
+    will all be merged together in a later step
+8.  When you are done and ready to start the process of making a pull
```

Comment:
Since this is an intro section, I wonder if it'd be better to skip steps 8-11 and just do them as part of the merge on github: https://help.github.com/articles/about-pull-request-merges/

Reply:
If I'm reading that right, it looks like the maintainer would do the squash; I think it's nicer to let people edit their own commit messages and choose what to say.  And also easier for us.  I can go either way if you thinks this reflects too much of my idiosyncratic workflow. :)

Reply:
Yep, you're reading it right.  I just get scared telling people new to git to use rebase, just so many ways it can go wrong.  Totally happy to leave this as is.